Tests/Resources/Fixtures/config/config2.xml

```diff
@@ -11,9 +11,12 @@
     <error enable-parent-provider="true" enable-sibling-provider="true" />
 
     <sitemap enabled="true">
-        <configuration default-change-frequency="never" name="default" guessers="none" voters="none" loaders="none">
+        <configuration default-change-frequency="never" name="default">
             <template format="xml">test.xml</template>
             <template format="html">test.html</template>
+            <loader>_none</loader>
+            <voter>_none</voter>
+            <guesser>_none</guesser>
```

Comment:
I would recommend to choose to use either attributes or elements, in this case (As it's a prototyped array) I would choose elements.

Otherwise, you have to write some conditionals (to not allow elements *and* attributes at the same time), which is not supported by PHP's DomDocument atm IIRC.